cmd/run/run.go

```diff
@@ -0,0 +1,83 @@
+package run
```

Comment:
having a package for a command should be fine, but I was just thinking if it is an overkill 🤔 

what is the reason for a command to have its own package?

Reply:
It allows easy navigation of commands. When the commands within commands will grow. As an ex. gen will multiple of these.